src/storage/errors.rs

```diff
@@ -385,7 +385,7 @@ pub fn extract_key_error(err: &Error) -> kvrpcpb::KeyError {
             key_error.set_retryable(format!("{:?}", err));
         }
         Error(box ErrorInner::Txn(TxnError(box TxnErrorInner::Mvcc(MvccError(
-            box MvccErrorInner::AlreadyExist { key },
+            box MvccErrorInner::AlreadyExist { key, .. },
```

Comment:
Are we gonna return it to TiDB as well, probably for logging?

Reply:
Yes I think so.